torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2703,6 +2703,22 @@ def generator():
     return list(generator())
 
 
+def sample_inputs_mvlgamma(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    tensor_shapes = ((S, S), ())
+    ns = (1, 2, 3, 4, 5)
+
+    def compute_min_val(d):
```

Comment:
This is going to need a comment; would you elaborate on what's going on here?

I'm surprised that sample_inputs_mvlgamma doesn't accept the `p` value as a kwarg - isn't it the `p` value that determines the allowable range?